api/src/unraid-api/cli/plugin.command.ts

```
@@ -0,0 +1,120 @@
+        if (!packageName) {
+            this.logService.error('Package name is required.');
+            process.exitCode = 1;
+            return;
+        }
+        try {
+            await this.dependencyService.addPeerDependency(packageName, options.bundled);
+            this.logService.log(`Added ${packageName} as a peer dependency.`);
+            if (!options.bundled) {
+                throw 'Feature disabled: Remote API plugins. Use --bundled to install a vendored plugin.';
+                await this.dependencyService.npmInstall();
+                await this.dependencyService.rebuildVendorArchive();
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)